docs/reference/configuration.txt

```diff
@@ -368,8 +368,11 @@ for details on driver options.
 
       # Maintain legacy behavior of pluck and distinct, which does not demongoize
       # values on returning them. Setting this option to false will cause
-      # pluck and distinct to return demongoized values.
-      # (default: false)
+      # pluck and distinct to return demongoized values. Setting this option to
+      # false will also allow passing _translations fields to pluck and distinct
+      # and will return embedded values themselves (i.e. without putting them
+      # in a hash).
+      # (default: true in Mongoid 7, will change to false in Mongoid 8)
```

Comment:
Since the default was already changed to false in master, I suggest leaving it as false here and in the 7.4 branch replacing false with the "true in 7, false in 8" line.

Reply:
done